components/pill/w-pill.vue

```diff
@@ -40,6 +53,12 @@ const closeClasses = computed(() => [
 </template>
 
 <script>
+import { activateI18n } from '../util/i18n';
+import { messages as enMessages} from './locales/en/messages.mjs';
+import { messages as nbMessages} from './locales/nb/messages.mjs';
+import { messages as fiMessages} from './locales/fi/messages.mjs';
+activateI18n(enMessages, nbMessages, fiMessages);
```

Comment:
I still find it a bit hard to understand the difference between script and script setup. Why all this cannot be inside <script setup>?